From 119231cbe89d726b1d2f1ef3b4a8f4135d981f89 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sat, 10 Jul 2021 22:59:54 +0200
Subject: [PATCH] status

---
 jim.org | 42 ++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 42 insertions(+)

diff --git a/jim.org b/jim.org
new file mode 100644
index 0000000000000000000000000000000000000000..e1d7cd2826d93e9d0f048d8f875c8b64af4386d8
--- /dev/null
+++ b/jim.org
@@ -0,0 +1,42 @@
+brand
+suse
+cloudposse
+børnene
+matematik
+mig selv sagde hunden
+monero
+der vil jeg gerne når jeg sidder med det
+det jeg kommer til at lave
+jeg håber at have noget klar til til uge32, men det afhænger af dig
+alting bliver bedre anden gang
+drykiss joker
+echo "hello, world\!"
+Eliminer risiko forlokumsaftalt
+få dit navn på den gode løsning
+wc | test <= 
+wc | test <= $(hvor-lang-er-en-kronik)
+the test off kiss
+Øver sig i at spille, rottefængeren
+Loke et ben i hver lejer old school devops bygger bro og får tingene til at ske hvem byggede regnbuen
+Sund fornuft, lad is snakke om det
+rabbit
+nødudgang exit strategi åben dør
+Mine vogn
+Danish dynamite
+SAFe
+Gurilla warfare
+F16
+Svinehund
+Fri tekst
+fri software- og open source-programmer
+fri krop og sind
+Håndværk
+Maksimal frihed skrive og udføre fysisk arbejde
+Maksimal frihed skrive og udføre fysisk arbejde ryge
+Min far var gør det selver, og jeg går også og roder med alt muligt både fysisk såvel som mentalt
+devout follower of
+egne opstillinger
+tænker arbejde i sin fritid eller fritid på arbejdet så lad os arbejde sammen på en åben og fri platform
+ophavretten til alle værker, fysiske såvel som imaterielle, udgivet eller udarbejdet under ansættelsen, tilfalder medarbejderen. sp længe det sker under en copyleft licens
+gamepad fysisk
+udvider rammen

From 5adf3847b32df43aa21d8285af4c61a0bfb8fe83 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sat, 10 Jul 2021 23:23:23 +0200
Subject: [PATCH] status

---
 jim.org | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/jim.org b/jim.org
index e1d7cd2826d93e9d0f048d8f875c8b64af4386d8..d5df01d67e8977c2819ced07559848bd0b385856 100644
--- a/jim.org
+++ b/jim.org
@@ -40,3 +40,5 @@ tænker arbejde i sin fritid eller fritid på arbejdet så lad os arbejde sammen
 ophavretten til alle værker, fysiske såvel som imaterielle, udgivet eller udarbejdet under ansættelsen, tilfalder medarbejderen. sp længe det sker under en copyleft licens
 gamepad fysisk
 udvider rammen
+jeg har udvidet rammen lidt, min far var gør det selver
+hvad gør vi med hockey

From 92a35baa78e350942c1cbd8a985c78feb2ad52c8 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 01:01:16 +0200
Subject: [PATCH] emne

---
 jim.org | 1 +
 1 file changed, 1 insertion(+)

diff --git a/jim.org b/jim.org
index d5df01d67e8977c2819ced07559848bd0b385856..5fd285ed5751a86414f4cb86813e1d0ef5f10b50 100644
--- a/jim.org
+++ b/jim.org
@@ -42,3 +42,4 @@ gamepad fysisk
 udvider rammen
 jeg har udvidet rammen lidt, min far var gør det selver
 hvad gør vi med hockey
+1. udkast til billag om rettigheder

From efc78875209b66559f97154698fee8c37c7ebda5 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 16:23:56 +0200
Subject: [PATCH] rettigheder

---
 jim.org | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/jim.org b/jim.org
index 5fd285ed5751a86414f4cb86813e1d0ef5f10b50..0ebebe6d2ab0e0b259357da4e1f4ffe400c3af7a 100644
--- a/jim.org
+++ b/jim.org
@@ -43,3 +43,11 @@ udvider rammen
 jeg har udvidet rammen lidt, min far var gør det selver
 hvad gør vi med hockey
 1. udkast til billag om rettigheder
+tysk dødsmetal band der hedder wasserværkswasser
+græd ikke over spildt vand
+I min fritid kan jeg gøre som jeg vil, og såfremt jeg udgivet det under en copyleft[1] licens, tilfaldet ophavsretten mig.
+Jeg kan ikke tage ikke copyleft kode og offentliggøre det somsådant, men jeg kan lade mig inspirere
+Sund fornuf, og bedste praksis bør altid benyttes
+Men ingen er hævet over et nasty hack
+Den skal muligvis poleres lidt, men jeg synes det holdet vand
+komma^

From 248276a217fd5e697b089bf5905db4f0d02098a9 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 16:27:38 +0200
Subject: [PATCH] Show me yours and I will show you min

Stoiskish
---
 jim.org | 1 +
 1 file changed, 1 insertion(+)

diff --git a/jim.org b/jim.org
index 0ebebe6d2ab0e0b259357da4e1f4ffe400c3af7a..f24bd4dbd082272473c402d01f52c90df9dd5450 100644
--- a/jim.org
+++ b/jim.org
@@ -51,3 +51,4 @@ Sund fornuf, og bedste praksis bør altid benyttes
 Men ingen er hævet over et nasty hack
 Den skal muligvis poleres lidt, men jeg synes det holdet vand
 komma^
+Det kommer til at foregå i det offentlige rum

From d1f5cdaaf8c42a4d545efbe90c01abdbabfa6823 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 18:39:33 +0200
Subject: [PATCH] historie

---
 jim.org | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/jim.org b/jim.org
index f24bd4dbd082272473c402d01f52c90df9dd5450..60db4e4443fdf66f8c0819f15926c29a2c9a59b2 100644
--- a/jim.org
+++ b/jim.org
@@ -52,3 +52,17 @@ Men ingen er hævet over et nasty hack
 Den skal muligvis poleres lidt, men jeg synes det holdet vand
 komma^
 Det kommer til at foregå i det offentlige rum
+livens ondskab
+check licensdn der kan gorekomme drager
+pandoc weawing
+1 for alle alle for en, mosketer ed
+Klaphat
+Kan mus lidt gullerødder? can'rotter
+Darkmarket
+cylender
+mega hummer hotdog
+den onde fe
+balder
+det kister spidsen af en jetjager
+den dybe tallerken
+the ice is melting at the poles

From 4159db74f96379d003df79cff9562eb5c27508f7 Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 19:59:29 +0200
Subject: [PATCH] Extended history

---
 jim.org | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/jim.org b/jim.org
index 60db4e4443fdf66f8c0819f15926c29a2c9a59b2..adf696bd7f51275ac738d0a7762a4c64dd06bd3c 100644
--- a/jim.org
+++ b/jim.org
@@ -66,3 +66,14 @@ balder
 det kister spidsen af en jetjager
 den dybe tallerken
 the ice is melting at the poles
+Det er agurketid
+Naivt at afskrive kammerateri som uvidenhed
+Nu er katten ude af pose
+Vand under broen
+Jeg aner ikke hvad der foregik, det var et stort cirkus
+Vi må passe på dyrene og deres habitat
+banging with the hammer
+eller måske dask-top hvad derikker møller
+Sprængful af gode ideer
+bacon vs ananas, ananas et mere oppe i tiden
+jeg er begyndt at lære levan polka, men den sidder ikke helt i skabet endnu

From 79c63c66d2d8d92da2f85e11f7c3f1bea72408bf Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 20:10:50 +0200
Subject: [PATCH] =?UTF-8?q?ikke=20flere=20sm=C3=B8ger,=20jeg=20m=C3=A5=20p?=
 =?UTF-8?q?=C3=A5=20det=20sorte=20marked?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 jim.org | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/jim.org b/jim.org
index adf696bd7f51275ac738d0a7762a4c64dd06bd3c..d377e9da9e7b329f3195241a73a1f0a80c176afc 100644
--- a/jim.org
+++ b/jim.org
@@ -77,3 +77,7 @@ eller måske dask-top hvad derikker møller
 Sprængful af gode ideer
 bacon vs ananas, ananas et mere oppe i tiden
 jeg er begyndt at lære levan polka, men den sidder ikke helt i skabet endnu
+Jeg er et menneske med en bred smag, der har prøvet lidt af hvert
+Jeg har brugt Jim som min lomme psykolog
+Jeg har brugt Jim fm. som min lomme psykolog
+Jeg har brugt Jim mf. som min lomme psykolog

From 22839f2057c8b0598056292140aeb2121271f9ae Mon Sep 17 00:00:00 2001
From: Kristian Nygaard Jensen <freeduck@member.fsf.org>
Date: Sun, 11 Jul 2021 22:12:38 +0200
Subject: [PATCH] Vi skal have det hele med

---
 jim.org | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/jim.org b/jim.org
index d377e9da9e7b329f3195241a73a1f0a80c176afc..9fd2366875e38f370510d9c26ce0a0591dc53dda 100644
--- a/jim.org
+++ b/jim.org
@@ -81,3 +81,6 @@ Jeg er et menneske med en bred smag, der har prøvet lidt af hvert
 Jeg har brugt Jim som min lomme psykolog
 Jeg har brugt Jim fm. som min lomme psykolog
 Jeg har brugt Jim mf. som min lomme psykolog
+Vi gør det forkert, jeg skriver dette brev tildels i afmagt og vanvitig arigsbab
+Jeg er ikke jurist, så jeg har brug for lidt hjælp
+Min plan var egentlig at vente til jul, og lave et revy nummer, men jeg er simpelthen så indebrandt
